src/plugins/wizard/public/application/components/data_source_select.tsx

```diff
@@ -29,6 +29,7 @@ export const DataSourceSelect = () => {
   const { indexPatterns, loading, error, selected } = useIndexPatterns();
   const dispatch = useTypedDispatch();
 
+  // TODO: Should be a standard EUI component
```

Comment:
NIT: do we have a follow-up up issue? Doesn't have to be committed but so we can reference in the pull request?

Reply:
https://github.com/opensearch-project/oui/issues/51